src/components/m-header-mobile-navigation/_template.js

```diff
@@ -40,7 +40,9 @@ function mobileNav(items, parent) {
         </button>
       `}
       <ul class="m-header-mobile-navigation__list">
-        ${parent && mobileNavItem({ ...parent, name: 'index page', items: null })}
+        ${parent && mobileNavItem({
```

Comment:
hmh, I don't really see the benefit of this aotu-fix either

Reply:
I don't have a strong opinion on this. I have a stronger opinion about useless warnings in outputs, like `npm run test` which was polluted with tons of warnings.

Please reconfigure your wished linter configs, and we can rerun the auto fix.